### PR TITLE
Add support for dual (co-op) mode in mania beatmap

### PR DESCRIPTION
**Pull request by andy840119**
image

support duel mode in mania beatmap.

### CODE REVIEW

**Comment by LeNitrous** [comment]
Shouldn't this be "Dual"?

**Comment by andy840119** [comment]
@LeNitrous co-op = dual in lazer

File: osu.Game.Rulesets.Mania/Beatmaps/ManiaBeatmapConverter.cs [hunk 1]
```diff
@@ -27,6 +27,7 @@ public class ManiaBeatmapConverter : BeatmapConverter<ManiaHitObject>
         protected override IEnumerable<Type> ValidConversionTypes { get; } = new[] { typeof(IHasXPosition) };
 
         public int TargetColumns;
+        public bool IsDuel;
```
Comment (LeNitrous) [inline osu.Game.Rulesets.Mania/Beatmaps/ManiaBeatmapConverter.cs]: No I mean this.

Comment (andy840119) [reply osu.Game.Rulesets.Mania/Beatmaps/ManiaBeatmapConverter.cs]: oops 
i'll change the name

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**